support/scripts/mkcpio

```diff
@@ -29,8 +56,9 @@ if [ -z "${CPIOTOOL}" ]; then
 	exit 1
 fi
 
+# Generate CPIO file
 cd "$2"
 find . -depth \( ! -iname ".gitkeep" \) -print \
 	| tac \
-	| "${CPIOTOOL}" -v -o --format newc > "${OUT_BASE}/${OUT_FILE}"
+	| "${CPIOTOOL}" ${FLAGS} -o --format newc > "${OUT_BASE}/${OUT_FILE}"
```

Comment:
To keep proper quoting of the individual arguments, you can then let bash expand the list variable with the `@` paramter:
```bash
"${CPIOFLAGS[@]}"
```